ocs_ci/ocs/ui/base_ui.py

```diff
@@ -343,7 +343,8 @@ def __init__(self, driver):
         self.ocp_version = get_ocp_version()
         self.ocp_version_full = version.get_semantic_ocp_version_from_config()
         self.page_nav = locators[self.ocp_version]["page"]
-        self.validation_loc = locators[self.ocp_version]["validation"]
+        if self.ocp_version_full != version.VERSION_4_11:
```

Comment:
Do we know, how it will be in the following OCP releases (4.12,...)? Or is this and the other changes tightly related to OCP 4.11 only and no lower or upper version?
Because if there is a chance, that it will apply also to the future releases, I would suggest to use `<` operator or similar (in the other conditions in this PR.

Reply:
+1

Reply:
I dont think we need to use "validation" dict in `page_nav` or `deployment_ui` classes.
So I deleted it